pkg/skaffold/build/bazel/dependencies.go

```diff
@@ -49,7 +48,7 @@ func GetDependencies(ctx context.Context, dir string, a *latestV1.BazelArtifact)
 
 	go func() {
 		<-timer.C
-		once.Do(func() { logrus.Warnln("Retrieving Bazel dependencies can take a long time the first time") })
+		once.Do(func() { log.Entry(ctx).Warn("Retrieving Bazel dependencies can take a long time the first time") })
```

Comment:
Do we need to append a new line at the end due to `Warnln` -> `Warn` change?

Reply:
This was a mistake but it shouldn't actually have any effect. The only difference in how the `ln` vs non `ln` logging functions work is how they append multiple arguments to one another. Since there's only one arg being passed here it shouldn't effect anything

Reply:
I did find a couple spots where I should've used `ln` but didn't, and fixed those ones. Thanks for pointing out 👍🏼